.github/workflows/build.yml

```diff
@@ -271,6 +271,10 @@ jobs:
         run: |
           cat ./build/meson-logs/meson-log.txt
 
+      - name: Generate media test files.
+        run: |
+            ./ci/generate-test-media.sh
```

Comment:
I think it should be called from meson as test prerequisite. After those changes it would be harder to test locally, because you need to generate files.

Or we even create test file repository, same as ffmpeg does with FATE.

Reply:
>Or we even create test file repository

Well this PR is trying to avoid having to do all that of course. Calling it in meson is probably better though yeah.

Reply:
I would prefer more unit test like testing, instead of black box ones, but I guess it can do.

Reply:
Moved to the meson file now.